AboutPage.html

```diff
@@ -0,0 +1,148 @@
+                        This year, we hope to expand our invitation list including organizations and individuals who
+                        want to work with us on shared projects that advance the cause of the Commons, free culture and
+                        open knowledge.
+                        So if you're active and engaged in the worlds of open content and knowledge - free software
+                        advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and
+                        foundations, lawyers, and activists - we hope you'll consider joining us this year to build a
+                        stronger, more vibrant commons together
+                </div>
+                <div class="contact_via_email">
+                    <div class="email_note">
+                        Please contact us per Email for nay further questions about Leading Eagle Brand!
+                    </div>
+                    <a href="mailto:dummytwiz@gmail.com" class="link_to_email">leadingeaglebrand@mail.co</a>
+                </div>
+
+            </section>
+
+        </div>
+        <div class="logo_container">
+            <div class="logo_inner_container">
+                <img src="/Assets/Logos/navlogo.png" alt="" class="logo_image">
+            </div>
+        </div>
+        <section class="event_sample">
+            <div class="sample_header">
+                <h5 class="text5">
+                    Our past events decorations
+                </h5>
+                <div class="decoration"></div>
+                <span class="text6">
+                    Take a look at the past two Leading Eagle which took place in Buenos Aires and in Warsaw
+                </span>
+            </div>
+            <div class="past_event_container">
+                <div class="pass_event_cards">
+                    <img src="/Assets/Images/img2-pe.jpg" alt="eventname" class="event_image">
+                    <div class="event_details">
+                        <h4 class="event-date">
+                            2021
+                        </h4>
+                        <span class="description">
+                            We've got the most frequent event in the past. It was one the bess event ever.
+                        </span>
+                    </div>
+                </div>
+                <div class="pass_event_cards">
+                    <img src="/Assets/Images/img3-pe.jpg" alt="eventname" class="event_image">
+                    <div class="event_details">
+                        <h4 class="event-date">
+                            2022
+                        </h4>
+                        <span class="description">
+                            We've got the most frequent event in the past. It was one the bess event ever.
+                        </span>
+                    </div>
+                </div>
```

Comment:
- [ ] Please try to align these two pictures in the center in order to match the design given 😉  Look at the picture below 👇 

Your design
![image](https://user-images.githubusercontent.com/90222110/235303441-db5b6f52-69ba-495d-bf26-38de1b8c95d0.png)